wd_core.au3

```diff
@@ -105,6 +105,7 @@ Global Enum _
 		$_WD_ERROR_SessionInvalid, _ ;
 		$_WD_ERROR_UnknownCommand, _ ;
 		$_WD_ERROR_UserAbort, _ ;
+		$_WD_ERROR_HtmlProcessing, _ ;

```

Comment:
You would also need to add a matching entry in $aWD_ERROR_DESC

Reply:
suplemented